instance/models/mixins/openedx_database.py

```diff
@@ -24,15 +24,18 @@
 import string
 
 from django.conf import settings
+from django.db import IntegrityError
 from django.template import loader
 from django.utils.crypto import get_random_string
 
 from .database import MySQLInstanceMixin, MongoDBInstanceMixin
+from .rabbitmq import RabbitMQInstanceMixin
```

Comment:
@bdero Can you use absolute imports? I know we still have a few relative ones (including the one right above) so you might have done this for consistency, but it would get us closer to be consistent to change the line above tp absolute, insteaad.